table/tables/tables_test.go

```diff
@@ -184,9 +184,11 @@ func (ts *testSuite) TestTypes(c *C) {
 	rs, err = ts.se.Execute(ctx, "select c1 + 1 from test.t where c1 = 1")
```

Comment:
ditto

Reply:
Do I need remove this line?

Reply:
no, I think you should rewrite the test in this way:
```go
rs = tk.MustQuery("select c1 + 1 from test.t where c1 = 1")
rs.Check()
```